runtime/services/ttsd/index.js

```diff
@@ -30,6 +30,8 @@ var LIGHTD_INTERFACE = 'com.rokid.light.key'
 
 var _TTS = null
 var _CONFIG = null
+var _isTtsConnected = false
```

Comment:
This variable is to identify if it is the first time to start, is it better to name _isTtsStartup?

Reply:
`_isTtsStartup ` is not as same as `_isTtsConnected` semantically. Value of `_isTtsConnected` should be opposite of `_isTtsStartup`.

Reply:
Good point!